admin/includes/functions/configuration_checks.php

```diff
@@ -21,7 +21,31 @@ function zen_validate_configuration_entry($variable, $check_string) {
      // check inputs - error should be a defined constant in the language files
      if (empty($data['error'])) return; 
      if (!defined($data['error'])) {
-        $error_msg = TEXT_DATA_OUT_OF_RANGE; 
+        $msg_array = explode('_', $data['error']);
+
+        $str_msg = array();
+
+        foreach ($msg_array as $key => $word) {
+          $str_msg[$key] = implode('_', array_slice($msg_array, 0, $key + 1));
+/*          if ($key == 0) {
+            $str_msg[$key] = $word;
+          } else {
+            $str_msg[$key] = $str_msg[$key - 1] . '_' . $word;
+          }*/
+        }
+
+        $str_count = count($str_msg);
+
+        switch (true) {
+          case ($str_count >= 3 && $str_msg[2] == 'TEXT_MIN_ADMIN'):
```

Comment:
I'm a dolt.

This is simply able to be performed by:
````php
case (strpos($data['error'], 'TEXT_MIN_ADMIN') === 0):
````

No need for `explode`, array walking and/or `implode`.

I believe I had some other potential reason for such additional capacity such as finding content within the string at other specified opportunities, but looking at this today, I'm not seeing the need for the extra work/code.